views/pages/get-involved/volunteer-application.pug

```diff
@@ -8,7 +8,7 @@ block body
     +section("white")
       form(action="/api/volunteer-application" method="post")
         label(for="first-name" class="required") First Name
-        input(id="first-name" class="text-input" name="first-name" required maxlength="100")
+        input(id="first-name" class="text-input" name="first-name" required maxlength="100" pattern=".*[^ ].*" oninvalid="this.setCustomValidity('Please enter your first name')" onchange="try{setCustomValidity('')}catch(e){}" oninput="setCustomValidity(' ')")

```

Comment:
what's the pattern doing? might be a good opportunity to add a comment

Reply:
done 👍  it checks for input that's not only whitespace